sanic/cli/app.py

```diff
@@ -125,9 +130,18 @@ def _get_app(self):
                 app_type_name = type(app).__name__
 
                 if not isinstance(app, Sanic):
+                    if hasattr(app, "__call__"):
```

Comment:
```python
if callable(app):
```

Reply:
sorted, I learnt something :) thank you!